src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1620,15 +1669,75 @@ boost::regex getColorEncodingRegex(bool isGitGrep)
    return regex;
 }
 
+
+Error Replacer::replacePreview(const size_t dMatchOn, const size_t dMatchOff,
+                               size_t eMatchOn, size_t eMatchOff,
+                               std::string* pEncodedLine, std::string* pDecodedLine,
+                               size_t* pReplaceMatchOff) const
+{
+   // if we're not encoded, we can avoid some logic
+   bool encoded = true;
+   if (eMatchOn == 0 && eMatchOff == 0)
```

Comment:
Is it not possible to determine the `encoded` status from the `encoding_` member when the `Replacer`  is constructed?

Reply:
Good catch, fixed this.